lib/elixir/test/elixir/test_helper.exs

```diff
@@ -1,4 +1,10 @@
-ExUnit.start [trace: "--trace" in System.argv]
+exclude =
+  case :erlang.system_info(:otp_release) do
```

Comment:
Alternately, we could use `:erlang.system_info(:version)` which returns the erts version, which does include information about point releases.